modules/post/linux/gather/enum_protections.rb

```diff
@@ -108,37 +111,33 @@ def check_hardening
     if selinux_installed?
       if selinux_enforcing?
         r = 'SELinux is installed and enforcing'
-        print_good r
-        report r
       else
         r = 'SELinux is installed, but in permissive mode'
-        print_good r
-        report r
       end
+      print_good r
+      report r
     end
 
     if yama_installed?
       if yama_enabled?
         r = 'Yama is installed and enabled'
-        print_good r
-        report r
       else
         r = 'Yama is installed, but not enabled'
-        print_good r
-        report r
       end
+      print_good r
+      report r
     end
   end
```

Comment:
Again this looks  _fine_ but it's a little weird to me that rubocop is changing the flow of these if/else statements